scripts/extensions/sams/src/notifications/index.ts

```diff
@@ -0,0 +1,30 @@
+import {superdeskApi} from '../apis';
+import {onSetCreated, onSetUpdated, onSetDeleted} from './sets';
+
+const websocketNotificationMap: {[key: string]: (event: any) => void} = {
```

Comment:
It would be good to have these callback functions strongly typed. i.e. add the following to the `interfaces.ts` file:

```ts
export interface ISAMSBaseEvent {
    // Every event from SAMS should contain the following
    user_id: string;
    session_id: string;
    extension: 'sams';

    // These attributes are provided with most events from SAMS
    item_id?: string;
    _etag?: string;
}

export type ISAMSWebsocketEvent = CustomEvent<ISAMSEvent>;
```

You can then use `ISAMSWebsocketEvent` in `websocketNotificationMap` and in the function signature of these callbacks (i.e. in `onSetCreated`, `onSetUpdated` and `onSetDeleted`)